en/FlyView/FlyView.md

```diff
@@ -1,61 +1,81 @@
-
 ## Video
-At the lower left of the display you will see video output. QGroundControl supports RTP and RTSP video streaming over your vehicles UDP connection. It also support directly connected UVC device support. More details on QGC Video support can be found on the [Video README](https://github.com/mavlink/qgroundcontrol/blob/master/src/VideoStreaming/README.md).
 
-By clicking on the video you can make it be the main display for the Fly view.
+At the lower left of the display you will see video output.
+*QGroundControl* supports RTP and RTSP video streaming over your vehicles UDP connection. 
+It also support directly connected UVC device support. 
+More details on QGC Video support can be found on the [Video README](https://github.com/mavlink/qgroundcontrol/blob/master/src/VideoStreaming/README.md).
+
+By clicking on the video you can make it be the main display for the *Fly View*.
 
 ## Instrument Panel
-To the right is an instrument panel showing you current information on your vehicle. The center section of the panel has multiple pages. You can switch between pages by clicking on the center section. 
 
-### Telemetry page
+To the right is an instrument panel showing you current information on your vehicle. 
+By default this displays vehicle telemetry, but you can switch the content using the drop down menu.
 
-<img src="../../assets/fly/InstrumentTelemetryPage.jpg" style="width: 100px;"/>
+The different content is listed below.
 
-The values shown within the telemetry page can be configured by clicking on the small gear icon.
+### Values (Telemetry)
 
-### Vehicle Health page
+The values page shows telemetry information.
+The displayed information can be configured by clicking on the small gear icon.
 
-<img src="../../assets/fly/InstrumentHealthPage.jpg" style="width: 100px;"/>
+![Instrument Page - for values/telemetry](../../assets/fly/instrument_page_values.jpg)
 
-This page shows you the health of the systems within your Vehicle. If any of the systems switchs from healthy to unhealthy this page will automatically be switched to.
 
-### Vibration Clipping page
+### Camera
 
-<img src="../../assets/fly/InstrumentClipPage.jpg" style="width: 100px;"/>
+The camera page can be used to manually trigger the camera.
 
-This page show you current vibration values and clip counts.
+![Instrument Page - for Camera](../../assets/fly/instrument_page_camera.jpg)
 
-## Guided Bar
-At the bottom of the view is the Guided Bar. The guided bar allows to to interact with your vehicle directly from the QGroundControl application. Options available vary by Vehicle and current Vehicle state. 
 
-Some of the possible options are:
+### Video Stream
 
-* Arm, Disarm, Emergency Stop
-* Takeoff
-* Change altitiude
-* Go to location
-* RTL
-* Pause
+This page is used to enable and record video streaming.
```

Comment:
FYI this is based on what you see when video is not enabled. If you want something else, can you provide screenshots?